content/en/community/hackathons/2023-03-athens/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Athens 2023
+date: 2023-01-20T07:20:00+02:00
+draft: false
+collapsible: true
+description: Information on the Athens Unikraft Hackathon, March 30-31, 2023
+image: /assets/imgs/2023-03-hack-athens.png
+---
+
+## Athens Unikraft Hackathon
+
+Unikraft with [Nubificus](https://nubificus.co.uk/), [the Institute of Communication and Computer Systems (ICCS)](https://www.iccs.gr/en/) and [the National Technical University of Athens (NTUA)](http://www.ntua.gr/index_en.html) come together to organize the **Unikraft Athens Hackathon** to be held on Thursday and Friday, March 30-31, 2023.
+
+The hackathon will take place as an in-person event at the Multimedia Amphitheatre at the Zografos campus of NTUA.
+Address is TODO
+TODO detailed instructions on reaching the location of the hackathon
+
+Support information and discussions will take place on [Discord](http://bit.ly/UnikraftDiscord) on the `#hack-athens23` channel.
+
+### Registration
+
+To take part in the hackathon please fill this [registration form](https://forms.gle/a315sJrzRQV8rZdz8) by **Sunday, March 19, 2023, 11pm CET**.
+
+### People
+
+The hosts of the hackathon are:
+
+* [Anastassios Nanos](TODO)
+* [Charalampos Mainas](TODO)
+* [Michalis Pappas](TODO)
+* [Panagiotis Kokkinos](TODO)
+* [George Goumas](TODO)
+* [Manos Varvarigos](TODO)
+* [Nectarios Koziris](TODO)
+
+As part of the Unikraft community [Răzvan Deaconescu](https://github.com/razvand/) (UPB) and [TODO](TODO) will be present onsite with other community members providing support online, on Discord.
+
+### Schedule
+
+#### Thursday, March 30, 2023
+
+| Time (CET)    | Session                                                              | Presenter          |
+| ------------- | -------------------------------------------------------------------- | ------------------ |
+| 10:00 - 10:15 | High-level presentation of unikernels and Unikraft                   |                    |
+| 10:15 - 11:15 | Introduction to Unikraft. Building and running standard applications |                    |
+| 11:15 - 11:30 | Break                                                                |                    |
+| 11:30 - 12:45 | Inside Unikraft: Building, configuring, using different libraries    |                    |
+| 12:45 - 14:00 | Lunch                                                                |                    |
+| 14:00 - 15:00 | Running (Complex) Application in Unikraft                            |                    |
```

Comment:
```suggestion
| 14:00 - 15:00 | Running (complex) applications in Unikraft                            |                    |
```

Reply:
This is correct. If parentheses were missing, you would write with capital `Complex`.